src/it/strip-tar/assembly.xml

```diff
@@ -0,0 +1,28 @@
+<assembly
+    xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3 http://maven.apache.org/xsd/assembly-1.1.3.xsd">
+    <id>tar</id>
+    <formats>
+        <format>tar</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <dependencySets>
+        <dependencySet>
+            <outputDirectory>/</outputDirectory>
+            <useProjectArtifact>true</useProjectArtifact>
+            <unpack>true</unpack>
```

Comment:
If unpack=false (which is the default value) I don't think we will have a reproducible build because the maven-assembly-plugin:single goal is called in the package phase, whereas the stripping of the jar file is done in the next phase (to be sure it is called after the jar file is produced). So in this case you would include in the tarball a JAR file that is not reproducible, and then the tar file will be be reproducible.